app/pages/patient/patient.js

```diff
@@ -160,7 +160,17 @@ var Patient = React.createClass({
   },
 
   renderPatientTeam: function() {
-    return this.transferPropsTo(<PatientTeam />);
```

Comment:
@jebeck I was actually on the fence. With that many props I see the argument that passing everything manually is tedious and verbose, but for me first looking at the code base I liked the explicitness of seeing exactly what was being passed. It also made it easier to try to find potential orphaned props. But I'm happy to change to spread if that is the desire going forward. They make quick mention of it in the docs here: [Manually Transferring Props](https://facebook.github.io/react/docs/transferring-props.html#manual-transfer)